index.html

```diff
@@ -179,53 +183,53 @@ <h1 class="project-name">Base Apparel coming soon page</h1>
             </li>
           </ul>
           <div class="fnl-btn">
-            <button class="btn" disabled>See project</button>
+            <button type="button" class="btn" disabled>See project</button>
           </div>
         </div>
       </section>
       <section class="visit">
         <h2 class="cnct">CONTACT US</h2>
-        <p class="note">If you have an application 
+        <p class="note">If you have an application
           you are interested in developing
-          , a feature that you need built 
+          , a feature that you need built
           or a project that needs coding.
-          I’d love to help with it 
-       </p>
-       <form class="prjct">
-         <input class="wrt" type="text" value="  Text input">
-         <input class="wrt" type="text" value="  Ushindi Dan">
-          <textarea class="textarea" name="comment">  Enter text here</textarea>
-       </form>
-       <div class="fnl-btn faa">
-        <button class="btn" disabled>See project</button>
-      </div>
+          I’d love to help with it
+        </p>
+        <form action="https://formspree.io/f/mgedkakg" method="POST" class="prjct">
+          <input class="wrt" maxlength="30" type="text" placeholder="Enter your name">
+          <input class="wrt" type="Email" placeholder="Enter your mail">
```

Comment:
It is not the best to receive emails after users submit the form, only to find out that the form is completed anonymously.

- [ ] Kindly make sure that you require as mandatory that the input fields are completed.